src/main/java/pl/gov/coi/pomocua/ads/accomodations/AccommodationsResource.java

```diff
@@ -79,7 +79,8 @@ public void delete(@PathVariable Long id) {
 
     @GetMapping("accommodations/{id}")
     public ResponseEntity<AccommodationOffer> get(@PathVariable Long id) {
-        return ResponseEntity.of(repository.findById(id).filter(BaseOffer::isActive));
+        AccommodationOffer accommodationOffer = repository.findById(id).filter(BaseOffer::isActive).orElseThrow(OfferNotFoundException::new);
+        return ResponseEntity.ok(accommodationOffer);
```

Comment:
It could be simplified, so that this method returns `AccommodationOffer` directly (without wrapping it into `ResponseEntity`).
`200 OK` is a default response for get endpoints anyway.

Reply:
Done